utils/qemu/patches/0001-configure-allow-disable-fortify_source.patch

```diff
@@ -11,9 +11,9 @@ OpenWrt base build system decide flavor of fortify_source to use
 
```

Comment:
@yousong This patch can not be upstreamed or rather could you cooperate with upstream, so they can look at it?
Would be nice to let them know about it.